Source/OxyPlot.Wpf.Shared/PlotViewBase.cs

```diff
@@ -373,10 +373,11 @@ protected void OnModelChanged()
         /// </summary>
         protected void Render()
         {
-            if (this.plotPresenter == null || this.renderContext == null || !(this.isInVisualTree = this.IsInVisualTree()))
```

Comment:
This raises questions too - it cannot throw a NullReferenceException. If it's just a refactoring - then the behavior is changed.

Reply:
This was mentioned in https://github.com/oxyplot/oxyplot/issues/1798#issuecomment-964202536 but looking briefly now it does look a bit odd.

I'll try to find time to take another look at this (I should have some tests lying around somewhere....); my WPF knowledge is pretty limited, so I'd welcome any input from you and @albtaeler.

Reply:
Didn't see the comment. Considering it, it's all right. The only question is - did this check play any role in performance?

Reply:
I believe the only reason for this check was that this.TransformToAncestor() fails with a NullReferenceException without my changes. As mentioned in https://github.com/oxyplot/oxyplot/issues/1798#issuecomment-964202536 I changed this behavior, because rendering it's not possible if it isn't in the VisualTree, but sometimes needed.
I could not see any performance changes. It is important that the property `this.isInVisualTree = this.IsInVisualTree();` is set.

Reply:
I was confused at this point only because I don't see a place where IsInVisualTree() could throw a NullReferenceException.